README.md

```diff
@@ -218,19 +220,41 @@ To start with the actual implementation, you may check our [IntelliJ Platform SD
 For those, who value example codes the most, there are also available [IntelliJ SDK Code Samples][gh:code-samples] and [IntelliJ Platform Explorer][jb:ipe] – a search tool for browsing Extension Points inside existing implementations of open-source IntelliJ Platform plugins.
 
 
+## Qodana integration
+
+To increase the project value, the IntelliJ Platform Plugin Template got integrated with [Qodana][docs:qodana], a code quality monitoring platform that allows you to check the condition of your implementation and find any possible problems that may require enhancing.
+
+Qodana brings into your CI/CD pipelines all the smart features you love in the JetBrains IDEs and generates an HTML report with the actual inspection status.
+
+Qodana inspections are accessible within the project on two levels:
+
+- using the [Qodana IntelliJ GitHub Action][docs:qodana-github-action], run automatically within the [Build](.github/workflows/build.yml) workflow,
```

Comment:
explain how to disable this

Reply:
Really? It's about removing a proper step from the workflow – that's rather obvious.

Reply:
assuming people know how to configure GH actions, then yes

Reply:
I don't want them to remove it. ;) If someone will decide to do that, they'll figure it out.